libs/kv/types.proto

```diff
@@ -20,6 +20,7 @@ option (gogoproto.testgen_all)  = true;
 message Pair {
   bytes key   = 1;
   bytes value = 2;
+  bool  index = 3;
```

Comment:
Pair is supposed to be an abstract type representing key-value pair. Adding the third field makes it a specific type. Therefore, we need to

a) make sure we don't use Pair anywhere else
b) rename it to IndexPair or something like that

Reply:
Also, what about `KI64Pair`? Do we need to change it too?

Reply:
> a) make sure we don't use Pair anywhere else
>  b) rename it to IndexPair or something like that

I did a reviewed and Pair is used outside of the index service. Therefore I created a new IndexPair instead of renaming it.

> Also, what about KI64Pair? Do we need to change it too?

I have no idea what it is and deleting it doesn't seem to break the test suite, so it is probably used nowhere.

Reply:
> Also, what about KI64Pair?

It was removed.

Reply:
> I did a reviewed and Pair is used outside of the index service.

where?

Reply:
Do we need kv.Pair, it was primarily used for indexing? Looking through the repo it seems we should remove it and have the new type implement the sorting interface.